### ISSUE TITLE
NXMHandler keeps resetting.

### ISSUE THREAD

**Issue by ArcadiaFell** [TanninOne/modorganizer#1018]
I could seem to download anything from the FO4 nexus into MO2. I proceeded to go and fix it with the NXMHandler, and saw that FO4 wasn't marked as a MO2 game. I fix that and then closed NXMHandler. I go and try to download and it still won't work. I go and see if I did some wrong with the handler, and I saw that it reset to the how it was before, just having Skyrim as the only game registered to MO2.
tl;dr, NXMHandler won't hold changes, making mod downloading impossible through MO2.

MO version is 2.0.7.
